app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -587,6 +588,7 @@ public String revisionsByFilename(String filename) throws IOException {
                     .param("meta", "notifications")
                     .param("notformat", "model")
                     .param("notwikis", "wikidatawiki|commonswiki|enwiki")
+                    //.param("notfilter","!read")
```

Comment:
uncomment

Reply:
I am sorry for that I did that for testing